tools/etc/frr/daemons

```diff
@@ -35,4 +35,5 @@ eigrpd=no
 babeld=no
 sharpd=no
 pbrd=no
+staticd=no
```

Comment:
If we are adding this here then a mechanism in tools/frr needs to be added to handle this situation.

Reply:
I think there are at least 2 files like that.
this file is just  a template that people can use as they want. no?